libraries/aws/provisioning/src/private/aws_iot_provisioning_internal.h

```diff
@@ -28,6 +28,9 @@
 #ifndef AWS_IOT_PROVISIONING_INTERNAL_H_
 #define AWS_IOT_PROVISIONING_INTERNAL_H_
 
+/* Standard include */
+#include <stdbool.h>
+
 /* The config header is always included first. */
 #include "iot_config.h"
```

Comment:
Doesn't seem like the config header is being included first. Are `bools` even used in this file?

Reply:
Yes, had removed them in the API layer PR, but somehow that didn't traverse here in the merge conflict.